ush/config.sh.3DRTMA_dev1

```diff
@@ -64,11 +64,11 @@ ADDNL_OUTPUT_GRIDS=( "hrrr" )
 TILE_LABELS="CONUS REGIONS"
 TILE_SETS="full NE,NC,NW,SE,SC,SW"
 
-DO_DACYCLE="true"
-#DO_RETRO="true"
-#LBCS_ICS_ONLY="true"
-DO_NONVAR_CLDANAL="true"
-DO_REFL2TTEN="false"
+DO_DACYCLE="TURE"
+#DO_RETRO="TURE"
+#LBCS_ICS_ONLY="TRUE"
+DO_NONVAR_CLDANAL="TURE"
+DO_REFL2TTEN="TRUE"
```

Comment:
For RTMA,  Line 71 should be 
DO_REFL2TTEN="FALSE"

Reply:
comment out line 71. This is the same as set it to FALSE.

Reply:
Thanks!  But it is still better to explicit set it to FALSE in case the default value may change. It is okay to leave it as is now.